dpgen/data/arginfo.py

```diff
@@ -185,13 +185,13 @@ def init_surf_jdata_arginfo() -> Argument:
             Argument("z_min", int, optional=True, doc=doc_z_min),
             Argument("vacuum_max", float, optional=False, doc=doc_vacuum_max),
             Argument("vacuum_min", float, optional=True, doc=doc_vacuum_min),
-            Argument("vacuum_resol", list, optional=False, doc=doc_vacuum_resol),
+            Argument("vacuum_resol", list[float], optional=False, doc=doc_vacuum_resol),
             Argument("vacuum_numb", int, optional=True, doc=doc_vacuum_numb),
             Argument("mid_point", float, optional=True, doc=doc_mid_point),
             Argument("head_ratio", float, optional=True, doc=doc_head_ratio),
-            Argument("millers", list, optional=False, doc=doc_millers),
+            Argument("millers", list[list[int]], optional=False, doc=doc_millers),
```

Comment:
Is it list[list[int]] or list[list[float]]?

Reply:
The former.